ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/LedgerDao.scala

```diff
@@ -200,17 +212,18 @@ trait LedgerWriteDao extends AutoCloseable with ReportsHealth {
   ): Future[Unit]
 
   /**
-    * Explicitly adds a new party to the list of known parties.
+    * Stores a party allocation of rejection thereof.
```

Comment:
```suggestion
    * Stores a party allocation or rejection thereof.
```
Or am I reading it wrong?

Reply:
Thanks. Already fixed in response to @SamirTalwar's review.